Tiltfile

```diff
@@ -61,12 +62,68 @@ def deploy_capi():
             if kb_extra_args:
                 patch_args_with_extra_args("capi-kubeadm-bootstrap-system", "capi-kubeadm-bootstrap-controller-manager", kb_extra_args)
 
+tilt_dockerfile_header_cso = """
+FROM ghcr.io/sovereigncloudstack/cso:v0.1.0-alpha.3 as builder
```

Comment:
Can the settings.get("cso_version") be used here?

Reply:
updated the same like this and it's working. 

```
tilt_dockerfile_header_cso = """
FROM ghcr.io/sovereigncloudstack/cso:{} as builder

FROM docker.io/library/alpine:3.18.0 as tilt
WORKDIR /
COPY --from=builder /usr/local/bin/helm /usr/local/bin/helm
COPY --from=builder /manager /manager
COPY .release/ /tmp/downloads/cluster-stacks/
""".format(settings.get("cso_version"))
```